### PR TITLE
Updated langs to include python-repl

### PR DESCRIPTION
**Pull request by zachgates**
Amend `codeblock._parsing.PY_LANG_CODES` to include valid `python-repl` lang code.

### CODE REVIEW

File: bot/exts/info/codeblock/_parsing.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 log = logging.getLogger(__name__)
 
 BACKTICK = "`"
-PY_LANG_CODES = ("python", "pycon", "py")  # Order is important; "py" is last cause it's a subset.
+PY_LANG_CODES = ("python", "python-repl", "pycon", "py")  # Order is important; "py" is last cause it's a subset.
```
Comment (Akarys42) [inline bot/exts/info/codeblock/_parsing.py:15]: Seeing the comment at the end of the line, `python-repl` should be before `python`. Have you tested the new regex?

**Review by Akarys42 — DISMISSED** [review]

File: bot/exts/info/codeblock/_parsing.py (hunk 1 above)
Comment (zachgates) [reply bot/exts/info/codeblock/_parsing.py:15]: I interpreted the bit about "subset" as referring uniquely to `"py"` -- being a subset of the other/prior elements. Reinserting `python-repl` at the beginning of the tuple, however, does not escape the regex, either.

**Review by Den4200 — APPROVED** [review]
Looks good to me, thanks!

**Review by Numerlor — CHANGES_REQUESTED** [review]
We should include all aliases from https://github.com/highlightjs/highlight.js/blob/cf63d2fc63c6fa0f7d1ddf99fcdbe8ebf8ee8b93/src/languages/python.js#L249

**Review by Senjan21 — APPROVED** [review]
Looks good! Thanks for the PR!

**Pull request merged**